### PR TITLE
Display Push author and partial date in classify notifications

### LINKED ISSUE #716: Add push author and push time to the notifications
This data can be useful to sheriffs to ping the author of the push and to identify where the push is in their Treeherder view.

### PR DESCRIPTION
**Pull request by EvaBardou**
Closes #716

### CODE REVIEW

File: mozci/console/commands/push.py [hunk 1]
```diff
@@ -49,6 +49,8 @@
 # Push {push.id} evolved from {previous} to {current}
 
 Rev: [{push.rev}](https://treeherder.mozilla.org/jobs?repo={branch}&revision={push.rev})
+Author: {push.author}
+Date: {date}
```
Comment (marco-c) [inline mozci/console/commands/push.py:53]:
```suggestion
Time: {date}
```

File: mozci/console/commands/push.py [hunk 2]
```diff
@@ -305,10 +307,14 @@ def _list_tasks(tasks):
             previous == PushStatus.BAD
             and current in (PushStatus.GOOD, PushStatus.UNKNOWN)
         ):
+            formatted_date = datetime.datetime.fromtimestamp(
+                push.date, tz=datetime.timezone.utc
+            ).strftime("Pushed at %H hours %M minutes %z")
```
Comment (marco-c) [inline mozci/console/commands/push.py:312]:
```suggestion
            ).strftime("%H:%M")
```

@Archaeopteryx is treeherder always using UTC timezone?
If so, we should always use UTC timezone here too

Comment (Archaeopteryx) [reply mozci/console/commands/push.py]: Makes sense if we cannot display it in the timezone used by the client.

Comment (Archaeopteryx) [reply mozci/console/commands/push.py]: Please also add the seconds to the timestamp, there can be more than one push per minute.

**Pull request merged**